src/App.tsx

```diff
@@ -1,19 +1,139 @@
 import React from 'react';
-import './App.css';
-
+import './App.scss';
+import todos from './api/todos';
 import users from './api/users';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+const todosCopy = todos.map(todo => ({ ...todo }));
+const prepareTodos = () => {
+  return todosCopy.map(todo => ({
+    ...todo,
+    user: users.find(user => user.id === todo.userId),
+  }));
+};
```

Comment:
just save to a new variable

```suggestion
const preparedTodos = todos.map(todo => ({
    ...todo,
    user: users.find(user => user.id === todo.userId),
  }));
```

Reply:
fix this comment from previous review